spec.html

```diff
@@ -41860,12 +41868,11 @@ <h1>Atomics.store ( _typedArray_, _index_, _value_ )</h1>
       <emu-alg>
         1. Let _buffer_ be ? ValidateIntegerTypedArray(_typedArray_).
         1. Let _indexedPosition_ be ? ValidateAtomicAccess(_typedArray_, _index_).
-        1. Let _arrayTypeName_ be _typedArray_.[[TypedArrayName]].
-        1. If _arrayTypeName_ is *"BigUint64Array"* or *"BigInt64Array"*, let _v_ be ? ToBigInt(_value_).
+        1. If _typedArray_.[[TypedArrayName]] is *"BigUint64Array"* or *"BigInt64Array"*, let _v_ be ? ToBigInt(_value_).
```

Comment:
While you're here could you please hoist `_elementType_` to the top and then use the IsBigIntElementType AO instead of casing on [[TypedArrayName]]?

Reply:
It looks like use of `IsBigIntElementType()` is mostly confined to cases where only an element-type alias is in scope, not a TypedArray; when a TypedArray alias is available, the usual idiom is:
`If _typedArray_.[[ContentType]] is ~BigInt~, then ...`
So I made that change instead.

Reply:
wfm, thanks!